show_words_in_document.py

```diff
@@ -1,6 +1,9 @@
 import data_access
 
+"""
+prints the indexed words in the database and their assigned relevances
+"""
 data_accessor = data_access.MySQLDataAccessor()
 for word in data_accessor.get_all_words():
-    print(word.text, word.relevance) #prints the indexed words and their assigned relevances
+    print(word.text, word.relevance) 
```

Comment:
Generally you don't want print statements in final products. If you are interested in a debug log, that's something else that I wouldn't worry about at this point.